test/rulesets/LinearAlgebra/symmetric.jl

```diff
@@ -41,4 +41,241 @@
         frule_test(f, (x, SymHerm(Δx, uplo)))
         rrule_test(f, ΔΩ, (x, ∂x))
     end
+
+    # symmetric/hermitian eigendecomposition follows the sign convention
+    # v = v * sign(real(vₖ)) * sign(vₖ)', where vₖ is the first or last coordinate
+    # in the eigenvector. This is unstable for finite differences, but using the convention
+    # v = v * sign(vₖ)' seems to be more stable, the (co)tangents are related as
+    # ∂v_ad = sign(real(vₖ)) * ∂v_fd
+
+    function _eigvecs_stabilize_mat(vectors, uplo)
+        Ui = Symbol(uplo) === :U ? @view(vectors[end, :]) : @view(vectors[1, :])
+        return Diagonal(conj.(sign.(Ui)))
+    end
+
+    @testset "eigendecomposition" begin
+        @testset "eigen/eigen!" begin
+            # avoid implementing to_vec(::Eigen)
+            asnt(E::Eigen) = (values=E.values, vectors=E.vectors)
+
+            function _eigen_stable(A)
+                F = eigen(A)
+                rmul!(F.vectors, _eigvecs_stabilize_mat(F.vectors, A.uplo))
+                return F
+            end
+
+            n = 10
+            @testset "eigen!(::$SymHerm{$T}) uplo=$uplo" for SymHerm in (Symmetric, Hermitian),
```

Comment:
Could you indicate that this only tests `frule`s, preferably in the testset label, but alternatively as a comment?